electron/ironfish/IronFishManager.ts

```diff
@@ -35,6 +35,8 @@ import sendMessageToRender from '../utils/sendMessageToRender'
 import { WorkerMessageType } from '@ironfish/sdk/build/src/workerPool/tasks/workerMessage'
 import { app, shell } from 'electron'
 
+export let ifNode: null | FullNode = null
```

Comment:
This is a messy way to expose the node to the tRPC server, but it works 🤷  Could clean this up if we decide to go with this.